ts/main.ts

```diff
@@ -1,65 +1,171 @@
+  'FC Barcelona':
+    'https://api.sportsdata.io/v4/soccer/scores/json/PlayersByTeamBasic/ESP/559?key=d96ee95088b9438aa8c07a785c81b6e9',
+  'Real Madrid CF':
+    'https://api.sportsdata.io/v4/soccer/scores/json/PlayersByTeamBasic/ESP/605?key=d96ee95088b9438aa8c07a785c81b6e9',
+  'Paris Saint-Germain FC':
+    'https://api.sportsdata.io/v4/soccer/scores/json/PlayersByTeamBasic/FRL1/600?key=d96ee95088b9438aa8c07a785c81b6e9',
+};
+
+async function loadTeams(): Promise<void> {
   try {
-    const response: Response = await fetch(apiUrl);
-    const data: Team[] = await response.json();
+    const teamsEPL = await fetchTeams(API_URLS.EPL);
+    const teamsESP = await fetchTeams(API_URLS.ESP);
+    const teamsFRL1 = await fetchTeams(API_URLS.FRL1);
 
-    const filteredTeams = filterTeams
-      ? data.filter((team: Team) => filterTeams.includes(team.Name))
-      : data;
+    const teams = [...teamsEPL, ...teamsESP, ...teamsFRL1];
 
     if (teamsContainer) {
-      filteredTeams.forEach((team: Team) => {
+      teamsContainer.innerHTML = '';
+      teams.forEach((team: Team) => {
         const teamElement: HTMLElement = document.createElement('div');
         teamElement.classList.add('team');
+        teamElement.innerHTML = `
+          <div class="team-logo">
+            <img src="${team.WikipediaLogoUrl}" alt="${team.Name}">
+          </div>
+          <div class="team-name"><p>${team.Name}</p></div>
+        `;
+        teamElement
+          .querySelector('.team-logo')
+          ?.addEventListener('click', () => {
+            console.log(`Team clicked: ${team.Name}`);
+            if (teamLogoElement) {
+              teamLogoElement.src = team.WikipediaLogoUrl;
+              teamLogoElement.alt = team.Name;
+            }
+            if (titleElement) {
+              titleElement.textContent = team.Name;
+            }
+            loadStarting11(team.Name);
+          });
+        teamsContainer.appendChild(teamElement);
+      });
+    }
+  } catch (error) {
+    console.error('Error loading teams:', error);
+  }
+}
 
-        const teamLogoElement: HTMLElement = document.createElement('div');
-        teamLogoElement.classList.add('team-logo');
-
-        const imgElement: HTMLImageElement = document.createElement('img');
-        imgElement.src = team.WikipediaLogoUrl;
-        imgElement.alt = team.Name;
-        teamLogoElement.appendChild(imgElement);
+async function loadStarting11(teamName: string): Promise<void> {
+  const apiUrl = starting11ApiUrls[teamName];
+  if (!apiUrl || !starting11Container) return;
 
-        const teamNameElement: HTMLElement = document.createElement('div');
-        teamNameElement.classList.add('team-name');
-        teamNameElement.innerHTML = `<p>${team.Name}</p>`;
+  try {
+    const response: Response = await fetch(apiUrl);
+    const players: Player[] = await response.json();
+    console.log(`Fetched players for ${teamName}:`, players);
+    starting11Container.innerHTML = '';
```

Comment:
Remove the console.log and you should remove a different way if needed without using `innerHTML`